sklearn/linear_model/tests/test_ridge.py

```diff
@@ -598,6 +598,34 @@ def test_ridgecv_store_cv_values():
     assert_equal(r.cv_values_.shape, (n_samples, n_responses, n_alphas))
 
 
+def test_ridge_classifier_cv_store_cv_values():
+    # Test RidgeClassifier
+    x = np.array([[-1.0, -1.0], [-1.0, 0], [-.8, -1.0],
+                  [1.0, 1.0], [1.0, 0.0]])
+    y = np.array([1, 1, 1, -1, -1]).reshape(-1, 1)
+
+    n_samples = x.shape[0]
+
+    alphas = [1e-1, 1e0, 1e1]
+    n_alphas = len(alphas)
+
+    r = RidgeClassifierCV(alphas=alphas, store_cv_values=True)
+
+    # with len(y.shape) == 2
```

Comment:
But this is the case with both tests. What do you intend to distinguish between the different ys?

Reply:
I was trying to replicate the test for RidgeCV: 

```python 
def test_ridgecv_store_cv_values():
    # Test _RidgeCV's store_cv_values attribute.
    rng = np.random.RandomState(42)

    n_samples = 8
    n_features = 5
    x = rng.randn(n_samples, n_features)
    alphas = [1e-1, 1e0, 1e1]
    n_alphas = len(alphas)

    r = RidgeCV(alphas=alphas, store_cv_values=True)

    # with len(y.shape) == 1
    y = rng.randn(n_samples)
    r.fit(x, y)
    assert_equal(r.cv_values_.shape, (n_samples, n_alphas))

    # with len(y.shape) == 2
    n_responses = 3
    y = rng.randn(n_samples, n_responses)
    r.fit(x, y)
    assert_equal(r.cv_values_.shape, (n_samples, n_responses, n_alphas))
```
However, for the first case `# with len(y.shape) == 1`I found that RidgeClassifierCV always returns cv_values wiht 3 dimensions. That's part of the confusion. 

I guess then two test are unnnecesary.

Reply:
Done! Removed it.